todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/TasksMvpController.java

```diff
@@ -38,38 +40,116 @@
  */
 public class TasksMvpController {
 
-    private final FragmentActivity mFragmentActivity;
+    public static final String ADD_EDIT_DIALOG_TAG = "ADD_EDIT_DIALOG_TAG";
 
-    // Null task ID means there's no task selected (or in phone mode)
-    @Nullable private final String mTaskId;
+    private final FragmentActivity mFragmentActivity;
 
+    @Nullable
     private TasksTabletPresenter mTasksTabletPresenter;
 
     private TasksPresenter mTasksPresenter;
 
+    //@Nullable
+    //private AddEditTaskFragment mAddEditTaskFragment;
+
```

Comment:
Commented code shouldn't be kept, at least with a detailed explanation about why that is there, being a reference project, could make sense with a reason.